extensions/azurecore/package.nls.json

```diff
@@ -25,5 +25,6 @@
 	"config.azureAuthMethodConfigurationSection": "Azure Authentication Method",
 	"config.azureCodeGrantMethod": "Code Grant Method",
 	"config.azureDeviceCodeMethod": "Device Code Method",
+	"config.noSystemKeychain": "Disable system keychain integration. This is a temporary option and will be removed in future releases.",
```

Comment:
The text shouldn't say this is temporary, since we'll have to make that call based on several factors.  When we deprecated it we can provide a runtime notification in the release prior to removal.  Also, is there a way to make this a Linux-only option since it doesn't make sense on Windows\macOs.